package.json

```diff
@@ -54,9 +54,15 @@
     }
   },
   "devDependencies": {
+    "babel-jest": "^23.0.1",
     "enzyme": "^3.1.0",
     "enzyme-adapter-react-16": "^1.0.2",
     "enzyme-to-json": "^3.1.4",
+    "eslint": "^4.19.1",
```

Comment:
These are the result of installing dependencies for `eslint-config-airbnb` through `npx install-peerdeps --dev eslint-config-airbnb`. This is the only way I could run lint.

Reply:
Same here, we've added `eslint` in `rc-tools`.

Reply:
I can remove them, but I couldn't commit without linting. Is there an install step I missed to get that working without the updates here?

Reply:
this is the error you get after a normal `npm i` and `npm run test`
```
rc-tools run lint
/Users/matt.lein/rc-input-number/node_modules/eslint/lib/config/config-file.js:332
            throw e;
            ^

Error: Cannot read config package: eslint-config-airbnb
Error: Cannot find module 'eslint-config-airbnb'
Referenced from: /Users/matt.lein/rc-input-number/node_modules/rc-tools/lib/eslintrc
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at loadPackage (/Users/matt.lein/rc-input-number/node_modules/eslint/lib/config/config-file.js:168:16)
    at loadConfigFile (/Users/matt.lein/rc-input-number/node_modules/eslint/lib/config/config-file.js:212:18)
    at load (/Users/matt.lein/rc-input-number/node_modules/eslint/lib/config/config-file.js:385:18)
    at /Users/matt.lein/rc-input-number/node_modules/eslint/lib/config/config-file.js:326:36
    at Array.reduceRight (<anonymous>)
    at applyExtends (/Users/matt.lein/rc-input-number/node_modules/eslint/lib/config/config-file.js:309:28)
```